iroha-lib/model/TxBatch.cpp

```diff
@@ -0,0 +1,42 @@
+#include "TxBatch.hpp"
+
+#include "model/converters/pb_common.hpp"
+
+
+namespace iroha_lib
+{
+
+iroha::protocol::Transaction::Payload::BatchMeta::BatchType TxBatch::getBatchType(bool atomic) const
+{
+    const auto atomic_batch_type = iroha::protocol::Transaction::Payload::BatchMeta::BatchType::Transaction_Payload_BatchMeta_BatchType_ATOMIC;
```

Comment:
`using` might be useful here. https://en.cppreference.com/w/cpp/language/using_declaration

Reply:
Added.